downloads.md

```diff
@@ -72,6 +61,4 @@ Once you've downloaded Spark, you can find instructions for installing and build
 
 As new Spark releases come out for each development stream, previous ones will be archived, but they are still available at [Spark release archives](https://archive.apache.org/dist/spark/).
 
-### Nightly Packages and Artifacts
```

Comment:
Very belated, but, we had discussed this a while ago and agreed with other ASF folks that this link was OK.  It seems fine to move it to 'contributing' (see http://www.apache.org/legal/release-policy.html#host-rc ) but seems like it was just deleted.